src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -194,6 +194,34 @@ export const CustomerInstitutionMetadataForm = ({
               )}
             </Field>
 
+            <Field name={CustomerInstitutionFieldNames.RboInstitution}>
+              {({ field }: FieldProps<boolean>) => (
+                <div>
+                  <FormLabel
+                    component="legend"
+                    sx={{
+                      marginTop: '-1rem',
+                      marginLeft: '2rem',
+                    }}>
```

Comment:
Ser i skissene at det er brukt bold font på nvi og rbo, så lan være greit å legge inn dette på alle `FormLabel` det gjelder.
```suggestion
                  <FormLabel
                    component="legend"
                    sx={{
                    fontWeight: 'bold',
                      marginTop: '-1rem',
                      marginLeft: '2rem',
                    }}>
```